modules/gdscript/gd_parser.cpp

```diff
@@ -2963,18 +2963,37 @@ void GDParser::_parse_extends(ClassNode *p_class) {
 	}
 
 	while (true) {
-		if (tokenizer->get_token() != GDTokenizer::TK_IDENTIFIER) {
 
-			_set_error("Invalid 'extends' syntax, expected string constant (path) and/or identifier (parent class).");
-			return;
-		}
+		switch (tokenizer->get_token()) {
+
+			case GDTokenizer::TK_IDENTIFIER: {
+
+					StringName identifier = tokenizer->get_token_identifier();
+					p_class->extends_class.push_back(identifier);
+				}
+				break;
```

Comment:
It looks like clang-format doesn't like the indentation style you used: https://travis-ci.org/godotengine/godot/jobs/294820735